opentech/apply/funds/admin_helpers.py

```diff
@@ -71,6 +71,25 @@ def queryset(self, request, queryset):
         return queryset
 
 
+class RoundStateListFilter(admin.SimpleListFilter):
+    title = 'state'
+    parameter_name = 'form-state'
+
+    def lookups(self, request, model_admin):
+        return (
+            ('open', _('Open')),
```

Comment:
@Parbhat I would like to set "open" as the default for this page. The user should then be able to select all and closed as well. Best way to do that.

Reply:
@frjo Included a commit for setting index URL with Open filter.

Reply:
Excellent, thanks!